formats/logger/i18n/bn.json

```diff
@@ -0,0 +1,63 @@
+{
+    "edit": "{{user|$1}} {{link|$2}}কে সম্পাদিত করেছে {{diffSize|$3}} {{diff|$4}} {{summary|$5}}",
+    "edit-post": "{{user|$1}} {{board|$4|$5}}এ {{link|Thread:$2|$3}}কে সম্পাদিত করেছে {{diffSize|$6}} {{diff|$7}}",
+    "edit-reply": "{{user|$1}}এ {{board|$4|$5}}এ {{link|Thread:$2|$3}}এর উত্তরকে সম্পাদিত করেছে {{board|$4|$5}} {{diffSize|$6}} {{diff|$7}}",
+    "new": "{{user|$1}} {{link|$2}} বানিয়েছে {{diffSize|$3}} {{summary|$4}}",
+    "new-post": "{{user|$1}} {{board|$4|$5}}তে {{link|Thread:$2|$3}} পোস্ট করেছে {{diffSize|$6}} {{summary|$7}}",
+    "new-reply": "{{user|$1}} {{board|$4|$5}}তে {{link|Thread:$2|$3}}তে উত্তর দিয়েছে {{diffSize|$6}} {{summary|$7}}",
+    "threadclose": "{{user|$1}} {{board|$4|$5}}তে {{link|Thread:$2|$3}}কে বন্ধ করেছে {{summary|$6}}",
+    "threadreopen": "{{user|$1}} {{board|$4|$5}}তে {{link|Thread:$2|$3}}কে আবার খুলেছে {{summary|$6}}",
+    "threadremove": "{{user|$1}} {{board|$4|$5}} থেকে {{link|Thread:$2|$3}}কে সরিয়েছে {{summary|$6}}",
+    "threadremove": "{{user|$1}} {{board|$4|$5}} থেকে {{link|Thread:$2|$3}}কে ডিলিট করেছে {{summary|$6}}",
```

Comment:
There are duplicate `threadremove` keys in both translations and the `threaddelete` key has been removed.

Reply:
Ah, alright I corrected it.

Reply:
I meant, the `threaddelete` key was replaced with a duplicate `threadremove` key but I guess a missing `threaddelete` key isn't an issue since threads are going away on the UCP anyways.